spec/services/cfe/create_capitals_service_spec.rb

```diff
@@ -100,7 +100,7 @@ def savings_amount_with_nil_current_account
     end
 
     def expected_payload_hash # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
-      {
+      payload = {
```

Comment:
These changes look like they've crept in from another PR

Reply:
No - I assign it to payload here because lower down I add elements in to the hash if they're not nil.  It's to fix a flickering test

Reply:
Ok, that makes sense 👍